gravitee-plugin-core/src/main/java/io/gravitee/plugin/core/internal/PluginImpl.java

```diff
@@ -31,6 +32,8 @@ public class PluginImpl implements Plugin {
     private URL[] dependencies;
     private boolean deployed = true;
 
+    private long timestamp;
```

Comment:
Should name the variable to understand what it is ? createdTimestamp? LoadedTimestamp? WDYT?

Reply:
Agreed